### PR TITLE
Add partials for commonly shared/ updated content

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/3411

### CODE REVIEW

File: astro/deployment-resources.md [hunk 1]
```diff
@@ -103,7 +103,8 @@ Astronomer Deployments run a single scheduler. You can configure your scheduler
 
     The Airflow components of your Deployment automatically restart to apply the updated resource allocations. This action is equivalent to deploying code and triggers a rebuild of your Deployment image. If you're using the Celery executor, currently running tasks have 24 hours to complete before their running workers are terminated. See [What happens during a code deploy](deploy-project-image.md#what-happens-during-a-project-deploy).
 
-:::info Alternative Astro Hybrid setup
+<details>
```
Comment (lzdanski) [inline astro/deployment-resources.md:106]: I love this new feature, it looks fantastic and really helps with the content organization

**Review by lzdanski — COMMENTED** [review]
After merge conflicts get addressed, will take another pass, but first look - everything is *great* links work, build works, new styling is fantastic.

**Comment by jwitz** [comment]
@lzdanski The conflicts have been resolved and I added one more `details` note as you recommended. This should be good to merge soon!

**Comment by lzdanski** [comment]
Pages to review in preview

- [ ] astro/configure-worker-queues.mdx
- [ ] astro/connect-aws.md
- [ ] deployment-resources.md
- [ ] kubernetespodoperator.md
- [ ] resource-reference-hosted.mdx
- [ ] astro/runtime-version-lifecycle-policy.mdx
- [ ] software/runtime-version-lifecycle-policy.mdx

the following pages look like just rel-link changes:

- astro-architecture.md
- astro-glossary.md
- celery-executor.md
- cli/astro-deployment-create.md
- cli/astro-deployment-worker-queue-create.md
- cli/astro-deployment-worker-queue-delete.md
- cli/astro-deployment-worker-queue-update.md
- deployment-file-reference.md
- deployment-health-incidents.md
- deployment-settings.md
- executors-overview.md
- feature-previews.md
- features.md
- hipaa-compliance.md
- manage-deployments-as-code.md
- manage-hybrid-clusters.md
- astro/release-notes.md
- software/documentation-overview.md
- software/manage-airflow-versions.md
- software/version-compatibility-reference.md

New partials
- worker-type-table.mdx
- runtime-lifecycle-table.mdx

**Review by lzdanski — APPROVED** [review]
LGTM! Checked the new content in preview and the rel-link changes look like they're formatted correctly.

**Pull request merged**